integration-test/codecept.json

```diff
@@ -0,0 +1,34 @@
+{
+  "output": "./output",
+  "helpers": {
+    "Puppeteer": {
+      "url": "http://localhost",
+      "show": false,
+      "waitForNavigation": "networkidle0",
+      "waitForAction": 500,
+      "chrome": {
+        "args": [
+          "--no-sandbox",
+          "--disable-setuid-sandbox"
+        ]
+      }
+    }
+  },
+  "include": {
+    "I": "./support/steps_file.js",
+    "TodoList": "./support/pages/TodoList.js"
+  },
+  "mocha": {},
```

Comment:
Why is this necessary? We're using Jest as a test runner, so this looks a bit off.

Reply:
codecept uses mocha. wouldn't want to change their internals.

Reply:
Ah ok. Is it possible to leave it out then or does it break, if left? It does not override anything here...?

Reply:
i can remove it